torch/_subclasses/fake_tensor.py

```diff
@@ -418,6 +435,7 @@ def __call__(
             dynamic_dims=dynamic_dims,
             constraint_dims=constraint_dims,
             memoized_only=memoized_only,
+            allocate_fresh_if_metadata_mutated=allocate_fresh_if_metadata_mutated,
```

Comment:
Can we get rid of `memoized_only` ? it's not even been used anywhere. 

In general, I thin we should be more vigilant on adding more arguments to this function

Reply:
bump on this